helm/prometheus-rules/templates/alerting-rules/inhibit.management-cluster.rules.yml

```diff
@@ -1,10 +1,13 @@
+{{- if eq .Values.managementCluster.provider.flavor "vintage" }}
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
   creationTimestamp: null
   labels:
     {{- include "labels.common" . | nindent 4 }}
+{{- if not .Values.mimir.enabled }}
```

Comment:
This file is only for vintage clusters; I guess we don't need the condition then?

Reply:
Well that's true but i'm not sure if phoenix will not add CAPI ones here